### ISSUE TITLE
Opacity is ignored in 

### ISSUE THREAD

**Issue by corey34** [niklasvh/html2canvas#1895]
Images with opacity are ignored. This was not an issue with v1.0.0-alpha.12

 * html2canvas version tested with:  html2canvas 1.0.0-rc.3
 * Browser & version: Chrome 74.0.3729.169
 * Operating system: Windows

**Comment by hakimio** [comment]
Try to see if it works with RC1. There were some major changes in RC2 which broke a lot of things.

**Comment by corey34** [comment]
Yep, RC1 works for opacity, thanks. I really liked that RC3 worked with transform rotate but I will have to wait until opacity is working. Thanks

**Comment by t-sch** [comment]
![Unbenannt](https://user-images.githubusercontent.com/8858830/80692394-c835fa00-8ad1-11ea-8c97-39401d586fd3.PNG)
Hi,
I can confirm that opacity still doesn't work correctly in RC5.
In a simple sample

    <div id="container" style="opacity: 0.5">
       <div id="inner1" style="opacity:1">
          <img src="red.png" />
       </div>
       <div id="inner2" style="opacity:0.25">
          <img src="red.png" />
       </div>
    </div>
 
the inner1 image is shown correctly with a resulting opacity of 0.5 while inner2 is shown with 0.25 instead of 0.125. So it looks like cascaded opacity is not calculated.
In RC1 this sample works as expected.

**Comment by chrisrollins65** [comment]
Still not fixed as of rc7. Opacity just doesn't work. I see there's a [PR open](https://github.com/niklasvh/html2canvas/pull/2465) to fix it, but there's been no movement in this repo since December last year...
